docs/UserGuide.md

```diff
@@ -106,6 +104,56 @@ Examples:
 
 *  `edit_o 3 q/20 r/` Edits the quantity of food for the 3rd order to 20 and clears all remarks
 
+### Finding a supplier : `find_supplier / find_s`
```

Comment:
Would it be better if we wrote this `find_supplier / find_s` as `find_supplier` / `find_s` instead so that its clear the its two alternative commands

Reply:
Ok.